lib/src/main/java/io/ably/lib/util/Crypto.java

```diff
@@ -220,7 +221,13 @@ private CBCCipher(CipherParams params) throws AblyException {
 			System.arraycopy(plaintext, 0, cipherIn, 0, plaintextLength);
 			System.arraycopy(pkcs5Padding[padding], 0, cipherIn, plaintextLength, padding);
 			System.arraycopy(getIv(), 0, ciphertext, 0, blockLength);
-			byte[] cipherOut = encryptCipher.update(cipherIn);
+			byte[] cipherOut = new byte[0];
```

Comment:
You don't need to assign anything to `cipherOut` here; just declare it.

Reply:
@paddybyers done https://github.com/ably/ably-java/pull/388/commits/864268a24b5482fd26406dc68bc7f778456bcbe7